fido/Crypto/Fido2/AttestationNew.hs

```diff
@@ -0,0 +1,25 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module Crypto.Fido2.AttestationNew
+  ( VerifiableAttestationStatementFormat (..),
+  )
+where
+
+import Control.Exception (Exception)
+import qualified Crypto.Fido2.Model as M
+import Data.Kind (Type)
+
+-- TODO: Document
```

Comment:
Probably good to document before we continue?

Reply:
Done that!